### PR TITLE
Implement /H3D/ELEM/FAILURE for QUAD element

### PR DESCRIPTION
**Pull request by vdavaze**
#### Description of the feature or the bug

Add the possibility to use the recently implemented /H3D/ELEM/FAILURE output on 2D QUAD elements 

#### Description of the changes

Did the same development as the one made for solids and shells.

### CODE REVIEW

File: engine/source/output/h3d/h3d_build_fortran/create_h3d_quad_scalar.F [hunk 1]
```diff
@@ -44,7 +45,7 @@ SUBROUTINE CREATE_H3D_QUAD_SCALAR(H3D_DATA,ID_QUAD_SCALAR,ID_INPUT,TEXT1,STEXT1,
 C-----------------------------------------------
       TYPE (H3D_DATABASE) :: H3D_DATA
       INTEGER ID_QUAD_SCALAR,ID_INPUT,STEXT1,SCOMMENT,LAYER,IR,IS,IT,IUVAR
-      INTEGER IPART(LIPART1,*)
+      INTEGER IPART(LIPART1,*),ID
```
Comment (sebastienVilleneuve) [inline engine/source/output/h3d/h3d_build_fortran/create_h3d_quad_scalar.F:48]: INTENT statement is missing

File: engine/source/output/h3d/h3d_results/h3d_quad_scalar.F [hunk 2]
```diff
@@ -86,7 +87,7 @@ SUBROUTINE H3D_QUAD_SCALAR(
      .   IGEO(NPROPGI,NUMGEO),INVERT(*), NV46,ID_ELEM(*),
      .   INFO1,INFO2,IS_WRITTEN_QUAD(*),IPARTQ(*),IPARTTG(*),H3D_PART(*),
      .   LAYER_INPUT ,IPT_INPUT,GAUSS_INPUT,PLY_INPUT,IUVAR_INPUT,
-     .   IR_INPUT,IS_INPUT,IT_INPUT
+     .   IR_INPUT,IS_INPUT,IT_INPUT,ID
```
Comment (sebastienVilleneuve) [inline engine/source/output/h3d/h3d_results/h3d_quad_scalar.F:90]: INTENT statement is missing

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
Hi @vdavaze 

I have found 2 missing INTENT statement.

Sébastien

File: engine/source/output/h3d/h3d_results/h3d_quad_scalar.F (hunk 2 above)
Comment (vdavaze) [reply engine/source/output/h3d/h3d_results/h3d_quad_scalar.F]: Hi @sebastienVilleneuve 

The missing INTENT attributes have been added :-) ! Thanks a lot.

File: engine/source/output/h3d/h3d_build_fortran/create_h3d_quad_scalar.F (hunk 1 above)
Comment (vdavaze) [reply engine/source/output/h3d/h3d_build_fortran/create_h3d_quad_scalar.F]: Hi @sebastienVilleneuve 

The missing INTENT attributes have been added :-) ! Thanks.

Comment (vdavaze) [reply engine/source/output/h3d/h3d_build_fortran/create_h3d_quad_scalar.F]: Hi @sebastienVilleneuve , 

The squash is done. Sorry for having forgotten this. 

Have a nice afternoon. 

Thanks.

**Review by sebastienVilleneuve — APPROVED** [review]

**Pull request merged**